test/SerialIOTest.cpp

```diff
@@ -1777,13 +1802,18 @@ fileBased_add_EDpic(ParticleSpecies &e, uint64_t const num_particles)
 
 inline void fileBased_write_test(const std::string &backend)
 {
+#ifdef _WIN32
+    WSADATA wsaData;
+    WSAStartup(MAKEWORD(2, 0), &wsaData);
+#endif
```

Comment:
This looks very scary and would need extensive user documentation (and inline documentation), no? It essentially changes the user API contract.

Reply:
This would probably be solved (a bit) by above suggestion of splitting `enum class Method{HOSTNAME}` into `enum class Method{POSIX_HOSTNAME, WINSOCKS_HOSTNAME, MPI_PROCESSOR_NAME}`, this way making the use of Winsocks explicit. Documentation still needed obviously, yeah.

Reply:
I've removed support for Winsocks-Hostnames entirely, and further renamed the identifiers to `posix_hostname` and `mpi_processor_name`, so that people know to initialize MPI before using the latter.